app/components/rl-dropdown.js

```diff
@@ -19,13 +19,13 @@ export default Component.extend({
 
     propagateClicks: true,
 
-    manageVisibility: on('didInsertElement', observer('isExpanded', function() {
+    style: computed('isExpanded', function() {
         if (this.get('isExpanded')) {
-            this.$().css('display', 'block');
+            return 'display: block';
         } else {
-            this.$().css('display', 'none');
+            return 'display: none';
         }
-    })),
+    }),
```

Comment:
I think we have to use `htmlSafe()` here to avoid warnings 🤔 

```js
return htmlSafe(`display: ${this.get('isExpanded') ? 'block' : 'none'}`);
```

Reply:
I wonder if it wouldn't be better to use `.hidden` css class (https://github.com/rust-lang/crates.io/blob/master/app/styles/app.scss#L321) instead of inlining styles.
Maybe in the future we'll want to have a strict Content Security Policy? 🙃

Reply:
hmm, maybe. but I would guess that things like e.g. `ember-animated` will make that hard anyways.

Reply:
I'm leaving it inline (w/ `htmlSafe`) since it already works. We can come back to it in another issue/PR.